geps/gep-1619/index.md

```diff
@@ -59,6 +59,68 @@ session affinity, as this design is expected to be addressed within a separate G
+| Gloo Gateway 2.0 | [Session Affinity / Sticky Sessions](https://docs.solo.io/gloo-edge/latest/installation/advanced_configuration/session_affinity/) | [Session Affinity / Sticky Sessions](https://docs.solo.io/gloo-edge/latest/installation/advanced_configuration/session_affinity/) |
+| Google Kubernetes Engine | [Session Affinity](https://cloud.google.com/load-balancing/docs/backend-service#session_affinity) | [Session Affinity](https://cloud.google.com/load-balancing/docs/backend-service#session_affinity) |
+| HAProxy Ingress | [Affinity](https://haproxy-ingress.github.io/docs/configuration/keys/#affinity) | N/A |
+| HAProxy Kubernetes Ingress Controller | [Session Persistence](https://www.haproxy.com/documentation/haproxy-runtime-api/reference/enable-dynamic-cookie-backend/#sidebar) | N/A |
+| Istio | [Strong Session Affinity](https://istio.io/latest/docs/reference/config/networking/destination-rule/#LoadBalancerSettings-ConsistentHashLB) | [Soft Session Affinity](https://istio.io/latest/docs/reference/config/networking/destination-rule/#LoadBalancerSettings-ConsistentHashLB) |
+| Kong | [Persistent Session](https://docs.konghq.com/hub/kong-inc/saml/configuration/#config-session_remember) | N/A |
+| Nginx (Proxy) | [Session Persistence](https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/#enabling-session-persistence) | [Session Persistence](https://docs.nginx.com/nginx/admin-guide/load-balancer/http-load-balancer/#enabling-session-persistence) |
+| Traefik | [Sticky Sessions](https://doc.traefik.io/traefik/routing/services/#sticky-sessions) | N/A |
+
+Visualizing the result for what implementations call "Strong Session Affinity" (aka Session Persistence), it's mostly
+inconclusive:
+
+```mermaid
+pie title  What Implementations Call "Strong Session Affinity"?
+    "Session Affinity" : 5
+    "Sticky Sessions" : 5
+    "Session Persistence" : 3
+    "Persistent Session": 1
+    "Strong Session Affinity": 1
+    "Affinity": 1
+```
+
+This GEP chooses to use "session persistence" as the preferred definition of strong session affinity (guaranteed) while
+reserving "session affinity" to mean weak session affinity (best-effort). This GEP also approves "sticky sessions" as an
+acceptable name for "session persistence". We selected these names because "session persistence" and "session affinity"
+share a symmetry that indicates to users that there is a relationship between them. The term "Persistence" implies a
+sense of strong consistency or recurring behavior, whereas "Affinity" carries a weaker connotation related to liking or
+attraction.
+
+**Accepted definitions of "strong session affinity":**
+
+- Session Persistence (preferred)
+- Sticky Sessions
```

Comment:
I'm not really sure it's helpful or useful to have multiple ways to refer to this concept in Gateway API. I think it may be useful to have docs that mention some common alternative names for what we refer to as "Session Persistence", but I'd rather just settle on one name for this concept within Gateway API and run with it.

Reply:
Yea that's a good point. Better to keep it simple with less options for folks to remember. Removed.